src/libtorch.cc

```diff
@@ -335,6 +335,7 @@ ModelState::ParseParameters()
               .c_str());
     }
 
+    // TODO Re-enable NvFuser once fixed
```

Comment:
Make sure we have ticket to track re-enabling NvFuser

Reply:
I am using the same ticket. It is DLIS-3560.